src/components/NavMenuItems.vue

```diff
@@ -41,6 +41,7 @@ $vertical-padding: $nav-height-small - $nav-padding-small;
 .nav-menu-items {
   display: flex;
   justify-content: flex-end;
+  align-items: center;
```

Comment:
Why did we add this? I am fine with, its just a pretty picky component, thats why I ask.

Reply:
It was meant to align the language toggle when viewport is small in the nav, but looks like it's not needed anymore.